homeassistant/components/cloud/__init__.py

```diff
@@ -405,3 +413,19 @@ async def _service_handler(service: ServiceCall) -> None:
     async_register_admin_service(
         hass, DOMAIN, SERVICE_REMOTE_DISCONNECT, _service_handler
     )
+
+
+async def _setup_log_handler(hass: HomeAssistant) -> FixedSizeQueueLogHandler:
+    fmt = (
+        "%(asctime)s.%(msecs)03d %(levelname)s (%(threadName)s) [%(name)s] %(message)s"
+    )
+    handler = FixedSizeQueueLogHandler()
+    handler.setFormatter(logging.Formatter(fmt, datefmt=FORMAT_DATETIME))
+
+    integration = await async_get_integration(hass, DOMAIN)
+    loggers: set[str] = {"snitun", integration.pkg_path, *(integration.loggers or [])}
```

Comment:
`snitun` should not be here. It should be added to the loggers key in the manifest, same with `acme`.